locals.tf

```diff
@@ -26,6 +26,7 @@ locals {
   jump_vm_subnet                        = local.create_jump_public_ip ? module.vpc.public_subnets[0] : module.vpc.private_subnets[0]
   nfs_vm_subnet                         = local.create_nfs_public_ip ? module.vpc.public_subnets[0] : module.vpc.private_subnets[0]
   nfs_vm_subnet_az                      = local.create_nfs_public_ip ? module.vpc.public_subnet_azs[0] : module.vpc.private_subnet_azs[0]
+  worker_vm_subnets                     = var.use_single_subnet_for_workers ? [module.vpc.private_subnets[0]] : module.vpc.private_subnets
```

Comment:
The name `worker_vm_subnets` needs to change. They are compute nodes. Also the flag. Why do we have it, can user not supply a single or a group of subnets here? Usage is very strange. Also if we did use the flag, are the subnets always pulled from the first private_subnets array?

Reply:
Keeping in line with K8s reference and AWS docs, `nodes` in stead of `worker_vm`, would be self explanatory.

Reply:
But then I see `workers_*` already in use, so probably `workers_subnets` should do.